SchedulerJobs/SchedulerJobs.Sds.UnitTests/Jobs/AnonymiseHearingsConferencesAndDeleteAadUsersJobTests.cs

```diff
@@ -33,6 +33,9 @@ public void Setup()
         [Test]
         public async Task Timer_should_log_message_all_older_conferences_were_updated_and_update_job_history()
         {
+            // Arrange
+            Logger.IsEnabled(Microsoft.Extensions.Logging.LogLevel.Information);
```

Comment:
I don't think this is needed in the tests - it just checks a value, and doesn't do anything with it

Reply:
removed

Reply:
I can see you've updated this particular job - can you do the others too?